pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -169,18 +178,21 @@ Activity:
       format: date-time
       nullable: true
       description: The date time the activity definition (NOT the instance) was last updated
+    isFollowup:
+      type: boolean
+      description: whether this activity is considered a "follow-up"
 Activity.Summary:
   type: object
   title: Activity Summary
   required:
     - activityCode
-    - activityDashboardName
-    - activityDescription
+    - activityType
+    - activitySubtype
     - activityName
+    - activityTitle
```

Comment:
What is the requirement for this change? This is not cosmetic, and results in a breaking change to the API.

Reply:
I'm not aware of a product/stakeholder requirement for this change. This was done as part of refactor for DDP-4378, which helps resolve bug DDP-4376.

Reply:
Tracked in DDP-4471. Resolving comment.